modules/apigee_edge_apidocs/src/Entity/ApiDocInterface.php

```diff
@@ -85,25 +85,4 @@ public function getCreatedTime() : int;
    */
   public function setCreatedTime(int $timestamp) : self;
 
-  /**
-   * Returns the API Doc published status indicator.
-   *
-   * Unpublished API Doc are only visible to restricted users.
-   *
-   * @return bool
-   *   TRUE if the API Doc is published.
-   */
-  public function isPublished() : bool;
```

Comment:
Why did you remove this? Isn't it a requirement for an API documentation entity have published/unpublished state?
You probably wanted to extend the `\Drupal\Core\Entity\EntityPublishedInterface` here.

Reply:
Yes, added EntityPublishedInterface.